### PR TITLE
Support additional fuels BEIS (bioenergy)

### PR DESCRIPTION
**Pull request by greenercode**
**Is this a new addition or an update to an existing factor?**

- [x] New

**What category or categories does it affect?**

Fuel

**What?**

Additional bioenergy fuel emission factors 

**Why?**

Biomethane, biogas, and biomass were not supported.

### CODE REVIEW

**Review by maryarbab — CHANGES_REQUESTED** [review]
Great addition! Why do we include both units for each EF? Also, BEIS has “out of scope” emissions tab for bioenergy that we might want to consider to add.

**Comment by greenercode** [comment]
> Great addition! Why do we include both units for each EF? Also, BEIS has “out of scope” emissions tab for bioenergy that we might want to consider to add.

@patstirling @maryarbab @AnastasiaLobanova  outside of scope ones include more than just bioenergy - should these be added in the same PR or in a subsequent one?

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -365,15 +365,26 @@ Energy,Fuel,fuel_type_avgas-fuel_use_aviation,Aviation fuel (avgas),kg-CO2e/t,31
...
 Energy,Fuel,fuel_type_biodiesel_me_tallow-fuel_use_na,Biodiesel ME (from tallow),kg-CO2e/L,0.16751,,BEIS,2021,GB,2021/07/23,Emission intensity of methyl ester (ME) biodiesel from tallow (animal fat). Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,fuel_combustion
+Energy,Fuel,fuel_type_biodiesel_me_from_tallow-fuel_use_na,Biodiesel ME (from tallow),kg-CO2e/kg,0.16751,,BEIS,2021,GB,2021/12/14,Emission intensity of biodiesel ME (from tallow). Bioenergy conversion factors should be used for the combustion of fuels produced from recently living sources (such as trees) at a site or in an asset under the direct control of the reporting organisation. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,fuel_combustion
```
[the remark this answers is not in the document]
Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv:375]: And also from was not used before, e.g. line above

[the remark this answers is not in the document]
Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv:375]: Is the value of this EF correct? bc the units are different from the one above but the value is the same (I assume the density of diesel is higher than 1...)

**Review by AnastasiaLobanova — CHANGES_REQUESTED** [review]

**Review by maryarbab — DISMISSED** [review]

**Review by AnastasiaLobanova — APPROVED** [review]

**Review by AnastasiaLobanova — APPROVED** [review]

**Pull request merged**